### PR TITLE
Add support for xc7s_csga225 and the Digilent CMOD S7 board.

### PR DESCRIPTION
**Pull request by bl0x**
Tested on the CMOD s7 hardware and works.

### CODE REVIEW

**Comment by bl0x** [comment]
I tried out openFPGALoader for the first time yesterday, because f4pga project discontinued using openocd. I must say, I'm very happy with it =)

Thanks for the great work!

**Comment by trabucayre** [comment]
Thanks for the PR!
LGTM
Just a small remark: **doc/boards.yml** must be updated too to add this new board

**Comment by bl0x** [comment]
Great! I've added the doc.

File: src/board.hpp [hunk 1]
```diff
@@ -115,6 +115,7 @@ static std::map <std::string, target_board_t> board_list = {
 	JTAG_BOARD("arty_z7_20",      "xc7z020clg400",  "digilent", 0, 0, CABLE_DEFAULT),
 	JTAG_BOARD("axu2cga",         "xczu2cg",        "",         0, 0, CABLE_DEFAULT),
 	JTAG_BOARD("basys3",          "xc7a35tcpg236",  "digilent", 0, 0, CABLE_DEFAULT),
+	JTAG_BOARD("cmod_s7",         "xc7s25csga225",  "digilent", 0, 0, CABLE_DEFAULT),
 	JTAG_BOARD("cmoda7_35t",      "xc7a35tcpg236",  "digilent", 0, 0, CABLE_DEFAULT),
```
Comment (trabucayre) [inline src/board.hpp:119]: maybe (if it's relevant) to follow  the same naming as cmoda7_35t ?

Comment (bl0x) [reply src/board.hpp:119]: In fact I followed the naming of the previously existing (also Digilent boards) arty_a7_*

Comment (trabucayre) [reply src/board.hpp:119]: True!

**Comment by trabucayre** [comment]
Applied to master!
Thanks

**Pull request merged**